### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by drabiter**

### CODE REVIEW

**Comment by andesyudanto** [comment]
:ok_hand:

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # kodok
 a kodok repository
 
-1st - test ***
+1st - test ***TTTTT
```
Comment (gde-vt) [inline README.md]: should be AAA

**Comment by gde-vt** [comment]
:ok_hand:

**Comment by andesyudanto** [comment]
:ok_hand:

**Comment by andesyudanto** [comment]
:ok_hand:

**Comment by gde-vt** [comment]
:ok_hand:

**Pull request merged**